### ISSUE TITLE
Build failed due to duplicated class with App Langa

### ISSUE THREAD

**Issue by jefpetko1** [talsec/Free-RASP-Flutter#26]
hey @talsec-app 

after following instructions for adding dev and release dependency of the lib, I've encountered duplicated class issue for both build types:

Duplicate class a.a.a.a found in modules jetified-Applanga-3.0.153-runtime (com.applanga.android:Applanga:3.0.153) and jetified-TalsecSecurity-Community-4.2.1-release-runtime (com.aheaditec.talsec.security:TalsecSecurity-Community:4.2.1-release)

For translations in app we are using [App Langa](https://www.applanga.com/)

app.gradle:
    debugImplementation (com.aheaditec.talsec.security:TalsecSecurity-Community:4.2.1-dev")
    implementation ("com.aheaditec.talsec.security:TalsecSecurity-Community:4.2.1-release")

gradle.properties:
    android.enableJetifier=true

any thoughts about this one? 

Any suggestion would be very helpful,
thanks in advance

**Comment by talsec-app** [comment]
Hello @jefpetko1 

What version of freeRASP are you using? Please, try to hardcode your version to `3.0.2` and let us know, if the issue persists.

```yaml
# Hardcoded version without "^" syntax
freerasp: 3.0.2
```

Kind regards
Talsec Team

**Comment by jefpetko1** [comment]
hey @talsec-app 

I've accidentally reported the issue here in Free-RASP-Flutter instead of reporting it in Free-RASP-Android repo, sorry for that

Your hint helped, the issue indeed was the freeRASP version. As mentioned in comment above, I've used 4.2.1-dev/release instead of  **4.2.3-dev/release**. I copied readme dependency part and didn't notice that there was newer release.

So with 4.2.1-release/dev version duplicated class issue is indeed solved

Thanks a lot for your fast feedback and lib improvements!
Regards
